v2/pkg/stratus/runner/runner.go

```diff
@@ -203,9 +211,9 @@ func (m *Runner) setState(state stratus.AttackTechniqueState) {
 	m.TechniqueState = state
 }
 
-// GetUniqueExecutionId returns an unique execution ID, unique per run of Stratus Red Team (not for each TTP detonated)
+// GetUniqueExecutionId returns an unique execution ID, unique for each runner instance
 func (m *Runner) GetUniqueExecutionId() string {
-	return providers.UniqueExecutionId.String()
```

Comment:
note: This used to be the global unique execution ID that's now properly encapsulated